### ISSUE TITLE
Why is there always a big version delay between official Brave and Portable Brave here?

### ISSUE THREAD

**Issue by pstein** [portapps/brave-portable#126]
As you can see here:

https://brave.com/latest/

The newest Brave version is already v1.45.118
But Portable Brave is still on v1.43

It should only be a click on a "generateportablebrave" script to create a corresponding Portable brave package.

Really surprising that Portable Brave is neglected very much

**Comment by GunGunGun** [comment]
Yeah, prob need to implement auto script to automate this process because it's tedious.

**Comment by tactus72** [comment]
I do get a little concerned because no security vulnerabilities are being patched, so it would be nice if these apps had automated updates. Portableapps.com has 15 browsers that update themselves, so it is a little concerning, but I understand all the Portapps development takes time. Hoping for automatic upgrades one day.

**Comment by espanafly** [comment]
see https://github.com/portapps/brave-portable/issues/134#issue-1640713578

**Comment by Chaython** [comment]
https://github.com/Chaython/Brave-Portable-Updater

**Comment by ioc2e3** [comment]
Really surprising that Portable Brave is again neglected very much
